docs/orchestration/nextflow/nextflow-overview.md

```diff
@@ -370,22 +370,22 @@ aws batch submit-job \
 After submitting a workflow, you can monitor the progress of tasks via the AWS Batch console.
 For the "Hello World" workflow above you will see five jobs run in Batch - one for the head node, and one for each `Channel` text as it goes through the `hello` process.
 
-For a more complex example, you can try the following, which will run the [RNASeq workflow](https://nf-co.re/rnaseq) developed by the [NF-Core project](https://nf-co.re/) against data in the [1000 Genomes AWS Public Dataset](https://registry.opendata.aws/1000-genomes/):
+For a more complex example, you can try the following, which will run a demo [RNASeq workflow](https://github.com/nextflow-io/rnaseq-nf) against data in the [1000 Genomes AWS Public Dataset](https://registry.opendata.aws/1000-genomes/):
 
 ```bash
 aws batch submit-job \
     --job-name nf-core-rnaseq \
     --job-queue <queue-name> \
     --job-definition nextflow-<nextflow-stack-namespace> \
-    --container-overrides command=nf-core/rnaseq,\
+    --container-overrides command=rnaseq-nf,\
 "--reads","'s3://1000genomes/phase3/data/HG00243/sequence_read/SRR*_{1,2}.filt.fastq.gz'",\
```

Comment:
The parameters for this workflow are optional, but if you want to provide them they are
* `--reads` : the raw reads, e.g. a pair of fastq files
* `--transcriptome`: the "reference" feature set to count reads against

see: https://github.com/nextflow-io/rnaseq-nf/blob/192f12bc3e7528c681a9f82ab5ffc6823ae5c470/main.nf#L34-L37

Reply:
I haven't tested any other workflows, but I wanted to get the Quick Start operational and change the QS workflow to align to the example provided by Sequera. I will follow up.